esp/src/src-react/components/Filter.tsx

```diff
@@ -1,5 +1,25 @@
 import * as React from "react";
-import { FormGroup, TextField, FormControlLabel, Checkbox, Dialog, DialogTitle, DialogContent, DialogActions, Button, MenuItem, TextFieldProps } from "@material-ui/core";
+import {
+    getTheme,
+    mergeStyleSets,
+    FontWeights,
+    IDragOptions,
+    IIconProps,
+    ContextualMenu,
+    DefaultButton,
+    PrimaryButton,
+    IconButton,
+    Checkbox,
+    Dropdown,
+    IStackStyles,
+    Modal,
+    Stack,
+    TextField
+} from "@fluentui/react";
```

Comment:
Not a big fan of listing imports this way as I rarley look at them (vscode / linter will manage adding and removing items 98% of the time) and F12 will bring you to a given item without having to search for it.